Messaging/Forward Messages to Email - Node.js/index.js

```diff
@@ -0,0 +1,48 @@
+require("dotenv").config();
+let sw = require("@signalwire/realtime-api");
+let Messaging = sw.Messaging;
```

Comment:
you should be able to do something like 

```js
const { Messaging } = require("@signalwire/realtime-api")
```

so you don't need the temporary `sw` variable

Reply:
ok. Suggestion incorporated.